src/lib/Poster.svelte

```diff
@@ -121,6 +125,7 @@
           {title}
         {/if}
       </h2>
+      <time>{year}</time>
```

Comment:
```suggestion
     {#if year}
        <time>{year}</time>
      {/if}
```

We should probably put an {#if} around this, so if it is `undefined`, it won't show.